ci/scripts/combine-dbs.sh

```diff
@@ -13,6 +13,8 @@ else
 fi
 DB_NAME=benchmark.tmp.db alembic upgrade head
 
+sqlite3 benchmark.tmp.db < ci/scripts/migrate.sql
```

Comment:
We're using alembic to manage DB migrations. See https://github.com/coiled/coiled-runtime/blob/main/README.md#changing-the-benchmark-schema for instructions (you can reuse your script, just paste the query in the python skeleton that's auto generated.)
See https://github.com/coiled/coiled-runtime/pull/338/files#diff-5d6d77339ddca1137203f45ebe646d7b9227c52acd07a22bf8af33786ab20692 for an example

Reply:
Changed to use alembic. Testing now